src/main/java/training/endava/playground/generics/Collections/Exercises.java

```diff
@@ -0,0 +1,83 @@
+package training.endava.playground.generics.Collections;
+
+import java.util.*;
+
+public class Exercises implements Comparator<Items> {
+    ArrayList<Items> list = new ArrayList<Items>();
```

Comment:
as a best practice, you should let the type of the field to be as abstract as you can, basically, you call methods just from the List interface on this field
so, make this field reference type List